Driverless AI/time-series-recipe-tutorial-retail-sales-forecasting/time-series-recipe-tutorial-retail-sales-forecasting.md

```diff
@@ -851,7 +864,58 @@ Import H2O’s training and test subsets of the Retail Store Forecasting dataset
+
+There are two options:
+
+Option 1: Trigger a Driverless AI experiment to be trained once the forecast horizon ends. A Driverless AI experiment will need to be re-trained every week.
+
+Option 2: Use Test Time Augmentation to update historical features so that we can use the same model to forecast outside of the forecast horizon.
+
+Test Time Augmentation refers to the process where the model stays the same but the features are refreshed using the latest data. In our Walmart Sales Forecasting example, a feature that may be very important is the Weekly Sales from the previous week. Once we move outside of the forecast horizon, our model no longer knows the Weekly Sales from the previous week. By performing Test Time Augmentation, Driverless AI will automatically generate these historical features if new data is provided.
+
+In Option 1, we would launch a new Driverless AI experiment every week with the latest data and use the resulting model to forecast the next week. In Option 2, we would continue using the same Driverless AI experiment outside of the forecast horizon by using Test Time Augmentation.
+
+Both options have their advantages and disadvantages. By retraining an experiment with the latest data, Driverless AI has the ability to possibly improve the model by changing the features used, choosing a different algorithm, and/or selecting different parameters. As the data changes over time, for example, Driverless AI may find that the best algorithm for this use case has changed.
+
+There may be clear advantages for retraining an experiment after each forecast horizon or for using Test Time Augmentation. Refer to this example to see how to use the scoring pipeline to predict future data instead of using the prediction endpoint on the Driverless AI server.
+
+Using Test Time Augmentation to be able to continue using the same experiment over a longer period of time means there would be no need to continually repeat a model review process. The model may become out of date, however, and the MOJO scoring pipeline is not supported.
+
+![test-time-augmentation-support-table](assets/test-time-augmentation-support-table.jpg)
+
+For different use cases, there may be clear advantages for retraining an experiment after each forecast horizon or for using Test Time Augmentation. In this notebook, we show how to perform both and compare the performance: [Time Series Model Rolling Window](https://github.com/h2oai/driverlessai-tutorials/blob/master/driverlessai_experiments/timeseries/walmart_timeseries_experiment/timeseries_model_rollingwindow.ipynb).
+
+## Triggering Test Time Augmentation
+
+To perform Test Time Augmentation, create your forecast data to include any data that occurred after the training data ended up to the dates you want a forecast for. The dates that you want Driverless AI to forecast should have missing values (NAs) where the target column is. Target values for the remaining dates must be filled in.
```

Comment:
@Castro38 it seems this sentence is incomplete: "The dates that you want Driverless AI to forecast should have missing values (NAs) where the target column is."

Reply:
@5675sp this is direct fromt the Documentation page. I don't believe the sentence is incomplete. It makes sense to me. In your target column, have NAs for your missing values. I do think it's worded weirdly but it makes sense.